README_API.md

```diff
@@ -0,0 +1,338 @@
+information in the meta entity including errorType and errorDetail
+entities. See example below for more detail.
+
+***The Response Entity***
+
+In the case of a successful response, there may be a response entity as
+well. The response entity contains additional information about the
+results of the request and will contain method specific response
+entities and attributes. 
+
+***Error Example***
+
+Error responses contain a type, code and errorDetail. The errorDetail
+entity contains a code and some short text that describes the issue in
+more detail. Errors **will not** contain a response entity. 
+
+
+```json
+{
+  "meta" : {
+    "errorType" : "validation", //Indicates general category of error
+    "code" : 500, //Will mirror the HTTP status code
+    "errorDetail" : {
+      "code" : "foo.some.issue", //Represents the particular failure scenario
+      "shortText" : "You have a foo related issue" //A short text describing that scenario
+    }
+  }
+}
+```
+
+***Success Example***
+
+Success responses always return with an HTTP status code of 200. In most
+cases (but not all), the success response will also contain a response
+entity with method specific attributes and entities.
+
+```json
+{
+  "meta" : {
+    "code" : 200
+  },
+  "response" : {
+    "foo" : 1,
+    "hello" : "world"    
+  }
+}
+```
+
+API Overview
+============
+
+The Data Reward API has 3 major functional areas; Network Based
+Subscriber Identification (optional), eligibility, and rewarding.
+
+API Details
+===========
+
+Network Based Subscriber Identification (Optional)
+--------------------------------------------------
+
+Network Based Subscriber Identification allows a sponsor to seamless
+identify a mobile subscriber. This proprietary technology works by
+placing a 1x1 pixel call on any of the sponsor's digital properties.
+When the 1x1 pixel call is invoked, the sponsor passes the Aquto portal
+their user identifier. If the subscriber is on cellular, the Aquto
+portal will automatically detect the operator network (Verizon, AT&T,
+etc.) and the specific subscriber on that network and associate the
+sponsor's user identifier with the subscriber identifier on the
+corresponding operator network.
+
+This call must be made from the client when the sponsor's user is
+interacting with the sponsor's digital properties, such as a mobile
+application or mobile website. Since it is made client side there is no
+need for authentication.
+
+***HTTP vs. HTTPS***
+
+The identification pixel will work properly over HTTP or HTTPS. However,
+some operators still rely on HTTP based technology to perform the
+identification of subscribers. If the pixel uses HTTP, then those
+operators will automatically be disqualified for network based
+subscriber identification. I.e.: Subscribers on those operators will not
+be identified.
+
+It is highly recommended that HTTP be used wherever possible.
+
+|<!-- -->| <!-- -->|
```

Comment:
Can probably drop this line so that padding is the same above and below. Same with the other places